src/core/hle/service/olsc/olsc.cpp

```diff
@@ -52,6 +52,14 @@ class OLSC final : public ServiceFramework<OLSC> {
         rb.Push(RESULT_SUCCESS);
     }
 
+    void GetSaveDataBackupSetting(Kernel::HLERequestContext& ctx) {
+        LOG_WARNING(Service_OLSC, "(STUBBED) called");
+        u64 backup_setting = 0;
```

Comment:
```suggestion
        constexpr u64 backup_setting = 0;
```

and maybe add a comment indicating what a setting of 0 indicates if known